src/scripts/policy/apply_blueprint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+# shellcheck disable=SC1083,SC1090,SC1091,2154
+# SC1083: This is literal.
+# SC1090: Can't follow non-constant source. Use a directive to specify location.
+# SC1091: Not following. Shellcheck can't follow non-constant source.
+# SC2154: "var is referenced but not assigned". These values come from an external file.
+#
+# This script will apply and publish a local blueprint as well as apply it to the target subscription
+
+set -e
+
+error_log() {
+  echo "${1}" 1>&2;
+}
+
+usage() {
+  echo "apply_blueprint.sh: Deploy a locally stored blueprint to a subscription"
+  error_log "usage: apply_blueprint.sh <subscription> <location> <blueprint_name>"
+}
+
+if [[ "$#" -lt 2 ]]; then
+    usage
+    exit 1
+fi
```

Comment:
the third argument `blueprint_name` isn't defaulted to anything -- should it be required?